desk/sur/chat.hoon

```diff
@@ -59,8 +59,7 @@
   ==
 ::
 ::  $essay: a chat message with metadata
-+$  essay  [memo:d %chat =kind]
-::  $kind: whether or not the chat is a system message
```

Comment:
We definitely want to, it is just that this PR was supposed to land quickly. Since we are still at it, I might as well go and dig it.

We will also want to handle bot authors in activities, but this would require an overhaul of %activity types and we decided to postpone it for now.